content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,88 @@
+  select an |MO|. Click the :guilabel:`Quality Alert` button at the top of the |MO| to open a quality
+  alert form in a new page.
+- Open the :menuselection:`Inventory` app, click the :guilabel:`# To Process` button on an inventory
+  order type card (Receipts, Delivery Orders, etc.), and then select an order. Click the
+  :guilabel:`Quality Alert` button at the top of the order to open a quality alert form in a new
+  page.
+- Open the :menuselection:`Shop Floor` module, and then select a work center from the navigation bar
+  at the top of the page. Then, click the :guilabel:`⋮ (three vertical dots)` button at the
+  bottom-right of a work order card to open the :guilabel:`What do you want to do?` menu. Select the
+  :guilabel:`Create a Quality Alert` option from this menu to open a quality alert in a pop-up
+  window.
+
+.. note::
+   Depending on how a new quality alert form is opened, certain fields on the form may already be
+   filled in. For example, if a quality alert is created from a work order card in the *Shop Floor*
+   module, the :guilabel:`Product` and :guilabel:`Work Center` fields will be pre-filled.
+
+After opening a new quality alert form, fill it out as follows:
+
+- Enter a short :guilabel:`Title` that summarizes the issue with the product.
+- If the product issue was discovered at a specific work center, select it from the :guilabel:`Work
+  Center` drop-down menu.
+- If the product is included in a specific picking order, select it from the :guilabel:`Picking`
+  drop-down menu.
+- In the :guilabel:`Team` field, select the quality team that is responsible for managing the
+  quality alert.
+- If a specific employee should be responsible for the quality alert, select them from the
+  :guilabel:`Responsible` drop-down menu.
+- In the :guilabel:`Tags` field, select any tags relevant to the quality alert from the drop-down
+  menu.
+- Use the :guilabel:`Root Cause` field to select the cause of the quality issue, if known.
+- Choose a :guilabel:`Priority` level by selecting a :guilabel:`⭐ (star)` number between one and
+  three. Quality alerts with higher priorities appear at the top of the *Quality Alerts* Kanban board
+  in the *Quality* app.
+
+At the bottom of the quality alert form are four tabs, which can be filled out as follows:
+
+- In the :guilabel:`Description` tab, enter a description of the quality issue.
+- If any actions should be taken to fix the affected product, enter them in the
+  :guilabel:`Corrective Actions` tab.
+- If any actions should be taken to prevent the issue from occurring again, enter them in the
+  :guilabel:`Preventive Actions` tab.
+- In the :guilabel:`Miscellaneous` tab, select the :guilabel:`Vendor` of the product, specify when
+  the alert was assigned to a quality team in the :guilabel:`Date Assigned` field, and enter an email
+  address that will be CC'd on emails about the quality alert, in the :guilabel:`Email cc` field.
```

Comment:
The line was just a lil too long (: Also minor thing about cc'd vs cc'ed :joy: I literally had to look this up because i never thought about it before.
![image](https://github.com/odoo/documentation/assets/23611711/5dce7b48-5b12-423b-b054-0916c7a4223b)

```suggestion
  the alert was assigned to a quality team in the :guilabel:`Date Assigned` field, and enter an
  email address that will be cc'ed on emails about the quality alert, in the :guilabel:`Email cc`
  field.
```